From 37f7a283e95deb02fb029a35aec1ddc1ec0ace7f Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?=E9=BB=84=E6=B6=9B?= <huangtao@rock-chips.com>
Date: Mon, 25 Apr 2011 10:47:05 +0800
Subject: [PATCH] Revert "rk29: disable VT/FRAMEBUFFER_CONSOLE, enable
 FB_EARLYSUSPEND, try to fix some time unable to wake up system bug, and
 faster wake up system"

This reverts commit 1b56ce984734b2f7c05fede31e8cc23070b86d6c.
Conflicts:

	arch/arm/configs/rk29_sdk_defconfig

Android need VT to show /initlogo.rle.
---
 arch/arm/configs/rk29_Winaccord_defconfig  | 2 +-
 arch/arm/configs/rk29_a22_defconfig        | 6 +++++-
 arch/arm/configs/rk29_ddr3sdk_defconfig    | 2 +-
 arch/arm/configs/rk29_malata_defconfig     | 2 +-
 arch/arm/configs/rk29_phonesdk_defconfig   | 2 +-
 arch/arm/configs/rk29_sdk_defconfig        | 6 +++++-
 arch/arm/configs/rk29_sdk_yaffs2_defconfig | 2 +-
 7 files changed, 15 insertions(+), 7 deletions(-)

diff --git a/arch/arm/configs/rk29_Winaccord_defconfig b/arch/arm/configs/rk29_Winaccord_defconfig
index 1a7a9a9ccb52..f310b3cdadc3 100644
--- a/arch/arm/configs/rk29_Winaccord_defconfig
+++ b/arch/arm/configs/rk29_Winaccord_defconfig
@@ -784,7 +784,7 @@ CONFIG_INPUT_GPIO=y
 #
 # Character devices
 #
-# CONFIG_VT is not set
+CONFIG_VT=y
 CONFIG_CONSOLE_TRANSLATIONS=y
 CONFIG_VT_CONSOLE=y
 CONFIG_HW_CONSOLE=y
diff --git a/arch/arm/configs/rk29_a22_defconfig b/arch/arm/configs/rk29_a22_defconfig
index 9174793227da..8bb7ac58d113 100755
--- a/arch/arm/configs/rk29_a22_defconfig
+++ b/arch/arm/configs/rk29_a22_defconfig
@@ -875,7 +875,11 @@ CONFIG_GS_MMA8452=y
 #
 # Character devices
 #
-# CONFIG_VT is not set
+CONFIG_VT=y
+# CONFIG_CONSOLE_TRANSLATIONS is not set
+CONFIG_VT_CONSOLE=y
+CONFIG_HW_CONSOLE=y
+# CONFIG_VT_HW_CONSOLE_BINDING is not set
 CONFIG_DEVMEM=y
 CONFIG_DEVKMEM=y
 # CONFIG_SERIAL_NONSTANDARD is not set
diff --git a/arch/arm/configs/rk29_ddr3sdk_defconfig b/arch/arm/configs/rk29_ddr3sdk_defconfig
index 429dae5b07eb..21aa3bdb8727 100644
--- a/arch/arm/configs/rk29_ddr3sdk_defconfig
+++ b/arch/arm/configs/rk29_ddr3sdk_defconfig
@@ -875,7 +875,7 @@ CONFIG_GS_MMA8452=y
 #
 # Character devices
 #
-# CONFIG_VT is not set
+CONFIG_VT=y
 CONFIG_CONSOLE_TRANSLATIONS=y
 CONFIG_VT_CONSOLE=y
 CONFIG_HW_CONSOLE=y
diff --git a/arch/arm/configs/rk29_malata_defconfig b/arch/arm/configs/rk29_malata_defconfig
index 5846bb5252a9..b17e870c5434 100644
--- a/arch/arm/configs/rk29_malata_defconfig
+++ b/arch/arm/configs/rk29_malata_defconfig
@@ -822,7 +822,7 @@ CONFIG_GS_MMA8452=y
 #
 # Character devices
 #
-# CONFIG_VT is not set
+CONFIG_VT=y
 CONFIG_CONSOLE_TRANSLATIONS=y
 CONFIG_VT_CONSOLE=y
 CONFIG_HW_CONSOLE=y
diff --git a/arch/arm/configs/rk29_phonesdk_defconfig b/arch/arm/configs/rk29_phonesdk_defconfig
index 4d99ab1212cc..76ef42f00f75 100755
--- a/arch/arm/configs/rk29_phonesdk_defconfig
+++ b/arch/arm/configs/rk29_phonesdk_defconfig
@@ -903,7 +903,7 @@ CONFIG_GS_MMA8452=y
 #
 # Character devices
 #
-# CONFIG_VT is not set
+CONFIG_VT=y
 CONFIG_CONSOLE_TRANSLATIONS=y
 CONFIG_VT_CONSOLE=y
 CONFIG_HW_CONSOLE=y
diff --git a/arch/arm/configs/rk29_sdk_defconfig b/arch/arm/configs/rk29_sdk_defconfig
index ebc8a3d3d147..f41fa86d80be 100755
--- a/arch/arm/configs/rk29_sdk_defconfig
+++ b/arch/arm/configs/rk29_sdk_defconfig
@@ -1053,7 +1053,11 @@ CONFIG_GS_L3G4200D=y
 #
 # Character devices
 #
-# CONFIG_VT is not set
+CONFIG_VT=y
+CONFIG_CONSOLE_TRANSLATIONS=y
+CONFIG_VT_CONSOLE=y
+CONFIG_HW_CONSOLE=y
+# CONFIG_VT_HW_CONSOLE_BINDING is not set
 CONFIG_DEVMEM=y
 CONFIG_DEVKMEM=y
 # CONFIG_SERIAL_NONSTANDARD is not set
diff --git a/arch/arm/configs/rk29_sdk_yaffs2_defconfig b/arch/arm/configs/rk29_sdk_yaffs2_defconfig
index a5808462b54b..a027478c6502 100644
--- a/arch/arm/configs/rk29_sdk_yaffs2_defconfig
+++ b/arch/arm/configs/rk29_sdk_yaffs2_defconfig
@@ -825,7 +825,7 @@ CONFIG_INPUT_GPIO=y
 #
 # Character devices
 #
-# CONFIG_VT is not set
+CONFIG_VT=y
 CONFIG_CONSOLE_TRANSLATIONS=y
 CONFIG_VT_CONSOLE=y
 CONFIG_HW_CONSOLE=y
-- 
2.35.3

